com.microsoft.mrtk.input/Visualizers/Visuals/Models/generic-controller.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Which files are these from https://github.com/immersive-web/webxr-input-profiles/tree/main/packages/assets/profiles? I don't see a `generic-controller` folder 🤔

Reply:
I renamed the folder. I these should be [generic-trigger-touchpad-thumbstick](https://github.com/immersive-web/webxr-input-profiles/tree/main/packages/assets/profiles/generic-trigger-touchpad-thumbstick). I can rename it back to the repo's original name.

Reply:
Done (it was https://github.com/immersive-web/webxr-input-profiles/tree/main/packages/assets/profiles/generic-trigger-squeeze-touchpad-thumbstick)